share/spice/cricket/cricket.js

```diff
@@ -0,0 +1,159 @@
+(function (env) {
+    "use strict";
+    var SCORECARD_ENDPOINT = 'https://query.yahooapis.com/v1/public/yql?q=select%20*%20from%20cricket.scorecard%20where%20match_id%3D{$1}&format=json&env=store%3A%2F%2F0TxIGQMQbObzvU4Apia0V0&callback=',
```

Comment:
@moollaza The problem is Yahoo rate limit the API calls and if we have too much requests then we might not be able to show live score.

We can consider other APIs in case we want to show live score.